File: Rest-Api-V3/market-data-public/funding-estimates/funding-estimates.js
const axios = require('axios');

async function fetchFundingEstimates(marketCode) {
  const url = `https://api.ox.fun/v3/funding/estimates?marketCode=${marketCode}`;

  try {
    const response = await axios.get(url);
    console.log('Funding Estimates:', response.data);
  } catch (error) {
    console.error('Error fetching funding estimates:', error.response ? error.response.data : error.message);
  }
}

// Example usage
fetchFundingEstimates('BTC-USD-SWAP-LIN'); 

File: Rest-Api-V3/market-data-public/funding-estimates/funding-estimates.py
import requests

def fetch_funding_estimates(market_code):
    url = f'https://api.ox.fun/v3/funding/estimates?marketCode={market_code}'

    try:
        response = requests.get(url)
        response.raise_for_status()  # Raise an error for bad responses
        print('Funding Estimates:', response.json())
    except requests.exceptions.RequestException as error:
        print('Error fetching funding estimates:', error)

# Example usage
fetch_funding_estimates('BTC-USD-SWAP-LIN')